episodes/server_resources.md

```diff
@@ -0,0 +1,211 @@
+
+:::{tip}
+Executing a piece of long-running code with a running kernel is not recognised as user activity on the server and so the process is killed after 1 hour. This is a rare use case of interactive computing on a hub, but this limit can be reconfigured for a particular hub upon request by contacting [2i2c support](#). 
+
+<!-- TODO: Add internal reference for contacting 2i2c support when written up. -->
+:::
+
+
+(episode-server:shutdown-server)=
+## Shutting down a server as a user
+
+A user should shut down their own server on the hub once they are finished with their work. This can be done manually by applying the following steps.
+
+1. (JupyterLab) Navigate to *File > Hub Control Panel*, or alternatively (JupyterLab and RStudio) navigate to the URL `https://<your-hub-url>.2i2c.cloud/hub/home`
+1. In the Hub Control Panel, click on the ![Stop My Server](../media/episodes/server_resources/shutdown-2.png) button
+
+:::{image} ../media/episodes/server_resources/shutdown-1.png
+:width: 100%
+:alt: Screenshot showing the File > Hub Control Panel menu option in the JupyterLab user interface.
+
+:::
+
+## Managing another user's server as a Hub Champion
+
+Hub Champions have access to an Admin panel from the Hub Control Panel (or the URL `https://<your-hub-url>.2i2c.cloud/hub/admin`).
+
+:::{image} ../media/episodes/server_resources/manage-1.png
+:width: 100%
+:alt: Screenshot showing the Admin panel available to Hub Champions.
+
+:::
+
+::::{caution}
+:::{epigraph}
+With great power comes great responsibility.
+
+-- Spiderman
+:::
+Performing administrative actions as a Hub Champion should be done with great care. Respect the privacy of others and think before you act.
+
+::::
+
+### Stop and restart a user's server
+
+A user's server may sometimes become unresponsive, e.g. issues stemming from self-installed software via `pip` and/or `conda`. Hub Champions can restore user servers to their default environment by stopping and restarting their server by clicking the ![Stop Server](../media/episodes/server_resources/manage-2.png) followed by the ![Start Server](../media/episodes/server_resources/manage-3.png) button next  to the user's name from the Admin panel.
+
+:::{note}
+Self-installed software packages on the hub are *temporary* and are removed each time a server is shut down to maintain a clean default software environment. Software is persisted between sessions by {ref}`episode-images`, which is covered in the next episode.
+
+:::
+
+### Access a user's server
+
+Hub Champions's can gain control of a user's server that is currently running by clicking the ![Access Server](../media/episodes/server_resources/manage-4.png) button next to the user's name from the Admin panel.
+
+From here a Hub Champion can navigate the server *as if they are the user themselves*, which is useful for reproducing and debugging issues.
```

Comment:
In the longer term if such a thing is possible, it would be cool to be able to provide some server environment that is broken in a predictable way, and ask trainee hub administrators to debug the problem as an exercise.

Reply:
That's a super nice suggestion – it would involve setting up a number of "broken" servers equal to the number of participants but I think it's possible. I've captured this idea in issue https://github.com/czi-catalystproject/hub-champion-training/issues/61